DependencyInjection/HttplugExtension.php

```diff
@@ -207,4 +216,25 @@ private function configureAuthentication(ContainerBuilder $container, array $con
                 ->addArgument(new Reference($authServiceKey));
         }
     }
+
+    /**
+     * Verify that Puli is installed
+     *
+     * @param ContainerBuilder $container
+     * @throws \Exception
+     */
+    private function verifyDiscoveryInstalled(ContainerBuilder $container)
+    {
+        $enabledBundles = $container->getParameter('kernel.bundles');
```

Comment:
i guess we need to fake some things for the unit tests here. we can then also test the error situation.